Program.cs

```diff
@@ -1,4 +1,15 @@
 ﻿using System;
-Console.WriteLine ("Hello, World!");
-
-   
\ No newline at end of file
+for (; ; ) {
+   Console.Write ("Enter the number:");
+   int.TryParse (Console.ReadLine (), out int number);

```

Comment:
You are not checking whether the input is a number or not.
If we give any string as input, it will output "It is a Palindrome".
So you need to validate the input and allow only numbers.